indico/htdocs/js/indico/modules/abstracts.js

```diff
@@ -132,6 +132,20 @@
         }).on('click', '#edit-reviewed-for-track-list', function(evt) {
             evt.preventDefault();
             $(this).closest('.i-timeline-item').find('.i-box-footer').toggleClass('hidden');
+        }).on('focus', '.new-comment textarea', function() {
+            var $commentForm = $(this).closest('form');
+            $commentForm.find('.form-group').show('fast');
+            $commentForm.removeClass('unfocused');
+        }).on('click', '.new-comment', function(evt) {
+            evt.stopPropagation();
+        });
+
+        $(window).on('click', function() {
```

Comment:
It seems like `setupAbstractPage` runs only once and it's not re-executed when reloading with AJAX (adding a review, editing a comment, ...)